tests/layer_tests/pytorch_tests/test_quantize.py

```diff
@@ -58,30 +58,38 @@ class TestQuantizePerChannelDequantize(PytorchLayerTest):
     def _prepare_input(self):
         return (np.array(5.00 * np.random.rand(5, 6, 7, 8) + 5.00, dtype=np.float32),)
 
-    @pytest.mark.parametrize("scales", [ 
-        np.array([1.0, 0.21, 0.62, 0.5], dtype=np.float32),
-        np.array([0.21, 0.62, 0.5, 1.0], dtype=np.float32),
-        np.array([0.62, 0.5, 1.0, 0.21], dtype=np.float32),
-        np.array([0.5, 1.0, 0.21, 0.62], dtype=np.float32),
-    ])
-    @pytest.mark.parametrize("zero_points", [
-        np.array([0, 4, 2, 1], dtype=np.int32),
-        np.array([0, 1, 2, 3], dtype=np.int32),
-        np.array([0, 0, 0, 0], dtype=np.int32),
-        np.array([-1, 0, -4, 5], dtype=np.int32),
+    @pytest.mark.parametrize("scale, zero_point, axis", [ 
+        [
+            np.array([1.0, 0.21, 0.62, 0.5, 0.74], dtype=np.float32),
+            np.array([0, -1, 2, -3, 4], dtype=np.int32),
+            0
+        ],
+        [
+            np.array([1.0, 0.62, 0.74, 0.11, 0.89, 0.32], dtype=np.float32),
+            np.array([0, 2, 4, -5, 6, -7], dtype=np.int32),
+            1
+        ],
+        [
+            np.array([1.0, 0.21, 0.62, 0.5, 0.11, 0.89, 0.32], dtype=np.float32),
+            np.array([0, -1, 2, -3, 4, -5, -7], dtype=np.int32),
+            2
```

Comment:
Does it work on CPU?

Reply:
No, marked with xfail. It was added a few commits back, the merge must've removed it for one second, but now it's back!